src/components/item-tag-manage/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { message } from 'antd';
+
+import Action from './action';
+import Filter from './filter';
+import ItemTagManageList from './list';
+import ItemService from '@src/services/Item';
+
+function ItemTagManage() {
+  const [filter, setFilter] = useState({});
+  const [selectedItemIds, setSelectedItemIds] = useState([]);
+
+  const handleChangeFilter = (
+    isReviewedCheck: boolean,
+    hasItemTagsCheck: boolean,
+    itemMinorTypeId?: number,
+    keyword?: string
+  ) => {
+    setFilter({
+      ...(isReviewedCheck ? { isReviewed: isReviewedCheck } : {}),
+      ...(hasItemTagsCheck ? { hasItemTags: hasItemTagsCheck } : {}),
+      ...(itemMinorTypeId ? { itemMinorTypeId } : {}),
+      ...(keyword?.length > 0 ? { keyword } : {}),
+    });
+  };
+
+  const handleChangeAction = async (
```

Comment:
```suggestion
  const handleChangeActionSubmit = async (
```

1. 이벤트핸들러의 네이밍에는 핸들링하는 이벤트를 나타내는 동사가 포함됩니다! (ex: onClick, onFocus, onMouseEnter, ... )

Reply:
1820deb

Reply:
2b05eb4

action click 핸들러를 ItemTagManageAction prop으로 넘겨주는 대신에 체크된 아이템 리스트를 넘겨주는 것으로 변경하였는데
괜찮은가요??